tests/test_database.py

```
@@ -0,0 +1,64 @@
+
+    def test_utc_now_returns_aware_datetime(self):
+        result = _utc_now()
+        assert result.tzinfo is not None
+
+    def test_utc_now_is_utc(self):
+        from datetime import timezone
+        result = _utc_now()
+        assert result.tzinfo == timezone.utc
+
+
+def _get_column_default(model_class, column_name: str) -> Any:
```

Ruff (0.15.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_get_column_default`

(ANN401)